sdks/go/tools.go

```diff
@@ -1,11 +1,11 @@
 // +build tools
 
-package go
+package go_sdk_tools
```

Comment:
`go` is a keyword, so it's not a valid identifier name (at least according to `gofmt`)